cli/cmd/component.go

```diff
@@ -180,7 +180,7 @@ func (c *cliState) LoadComponents() {
 						// Parse all global CLI flags provided (filtered) by the user, then run the global
 						// CLI init function to initialize our logger, api client, and other global config
 						err := cmd.Flags().Parse(filteredCLIFlags)
-						initConfig() // @afiune NOTE we purposely run this func first and then check the err
```

Comment:
initConfig() has already been run by the persistent pre run on root; I don't see why it has to be run again.

Running it again has the undesirable side-effect of clearing out cli.Token.

Reply:
We need this if-not none of the CLI global flags will be parsed.

Reply:
Instead of removing `initConfig()` add this on line `189`
```go
if err := cli.NewClient(); err != nil {
	return err
}
```